tox.ini

```
@@ -0,0 +1,28 @@
+[tox]
+envlist = python{3.7,3.8,3.9,3.10}
```

[nits] I often see the expressions like below:

```suggestion
envlist = py{37,38,39,310}
```

The merit of this is less typing on command line like:
```
tox -e py39
```

---

I also prefer the `py`[ver].

but I encountered the following error while developing it / running it -

```bash
ERROR:  py37: InterpreterNotFound: python3.7
```

And changing it to `python`[ver] solved it.

See also here -
https://stackoverflow.com/questions/63180349/error-py37-interpreternotfound-python3-7-when-running-tox-from-github

---

Oh, I didn't know this! I understood. Thanks!